cmd/cloud/server.go

```diff
@@ -136,6 +137,13 @@ var serverCmd = &cobra.Command{
 			os.Setenv(model.MattermostChannel, mattermostChannel)
 		}
 
+		utilitiesGitURL, _ := command.Flags().GetString("utilities-git-url")
+		if utilitiesGitURL != "" {
+			model.SetUtilityDefaults(utilitiesGitURL)
+		} else {
+			return errors.New("utilities-git-url must be set")
+		}
+
```

Comment:
Can be slightly simplified 😉 
```suggestion
		if utilitiesGitURL == "" {
			return errors.New("utilities-git-url must be set")
		}
		model.SetUtilityDefaults(utilitiesGitURL)

```